### PR TITLE
Updated Python examples for PEP8 compliance

### PR DESCRIPTION
**Pull request by dave3d**

### CODE REVIEW

File: Examples/FilterProgressReporting/FilterProgressReporting.py [hunk 1]
```diff
@@ -15,57 +15,60 @@
...
-if len ( sys.argv ) < 4:
-    print( "Usage: "+sys.argv[0]+ " <input> <variance> <output>" )
-    sys.exit ( 1 )
+if len(sys.argv) < 4:
+    print("Usage: " + sys.argv[0] + " <input> <variance> <output>")
+    sys.exit(1)
 
-##! [python director command]
+
+# #! [python director command]
```
Comment (zivy) [inline Examples/FilterProgressReporting/FilterProgressReporting.py:31]: What does this comment mean? Are these directives for sphinx? The directives appear in this file in other places too.

File: Examples/AdvancedImageReading/AdvancedImageReading.py [hunk 2]
```diff
@@ -15,16 +15,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-#=========================================================================
+# =========================================================================
 
 from __future__ import print_function
```
Comment (zivy) [inline Examples/AdvancedImageReading/AdvancedImageReading.py:20]: Possibly remove all of the `from __future__ import print_function` as we no longer support Python 2.7. Probably in all examples.

File: Examples/FilterProgressReporting/FilterProgressReporting.py (hunk 1 above)
Comment (blowekamp) [reply Examples/FilterProgressReporting/FilterProgressReporting.py:31]: I think the ##! mark up may be important to sphinx.

Comment (dave3d) [reply Examples/FilterProgressReporting/FilterProgressReporting.py:31]: Oh, ok.  I can undo that.  pycodestyle says there should be a space, but we can ignore that.

**Comment by blowekamp** [comment]
Is there a configuration file for the style that should be check in too?

**Comment by zivy** [comment]
Possibly run [flake8](https://flake8.pycqa.org/en/latest/) as part of the CI?

**Comment by dave3d** [comment]
I did use any configuration file.  I just ran pycodestyle and then fixed things via pycharm and by hand.

**Pull request merged**